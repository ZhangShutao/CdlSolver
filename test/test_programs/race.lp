marathon :- &c{marathon}.
relay :- &c{relay}.
:- marathon, relay.