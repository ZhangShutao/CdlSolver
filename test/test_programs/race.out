stable model: 1
X:marathon;
A:marathon;
stable model: 2
X:relay;
A:relay;
Satisfiable